Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Demolisher.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>Alpha Animals</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[defName="AA_Demolisher"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Vehicle</bodyShape>

```

Comment:
Since the description says this creature is based upon the body shape of a titan beetle, I may consider swapping this to quadrupedelow, which is what we use for the vatgrown titan beetle added by VFE - Insects.